ReactComponents/UserView.js

```diff
@@ -91,10 +91,21 @@ var UserView = React.createClass({
       .then((response) => response.json())
       .catch((error) => this.catchError(error))
       .then((responseData) => {
+        // This was added here -- https://github.com/DoSomething/LetsDoThis-iOS/pull/853#discussion_r54018442
+        // If we turn on airplane mode and load this view, the catchError executes above 
+        // but this then block still executes. feels like i'm doing something wrong here.
         if (!responseData) {
```

Comment:
Also documents weirdness of why we need this (https://github.com/DoSomething/LetsDoThis-iOS/pull/853#discussion_r54018442)